src/System.Text.Json/src/System/Text/Json/Utf8JsonReader.TryGet.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Buffers.Text;
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    public ref partial struct Utf8JsonReader
```

Comment:
Why is this file being included again? We already have most of it here:
https://github.com/dotnet/corefx/blob/0ae011a57253959b79c952219c5fa16389255b7b/src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs#L22

We should delete this duplicate/out-of-date copy.

Reply:
Not sure. Deleted.